fetch-repos.sh

```diff
@@ -32,7 +32,7 @@ QONNX_COMMIT="d72fb8cf1608e87b07c28ff165943016f78dcc38"
 FINN_EXP_COMMIT="0724be21111a21f0d81a072fccc1c446e053f851"
 BREVITAS_COMMIT="b106358c4169d8a9b68cb2a531aa795417d74887"
 CNPY_COMMIT="8c82362372ce600bbd1cf11d64661ab69d38d7de"
-HLSLIB_COMMIT="a19482ba6886f6f26aff11b10126a82ce0dd7ab1"
+HLSLIB_COMMIT="1dd4743436ca0346ab54dd88edb456678a9eee25"
```

Comment:
Currently this PR points to a finn-hlslib branch, when this PR (https://github.com/Xilinx/finn-hlslib/pull/164) is merged, then the commit hash can be updated and this PR be merged.

Reply:
It now points to finn-hlslib dev, which contains: https://github.com/Xilinx/finn-hlslib/pull/165